doc/data/messages/t/too-many-lines/pylintrc

```diff
@@ -0,0 +1,2 @@
+[FORMAT]
+max-module-lines = 20
```

Comment:
```suggestion
max-module-lines = 10
```

Reply:
You do realize that would cause the good.py example to have an error as well.

Reply:
We can do that too : https://github.com/PyCQA/pylint/pull/8203#discussion_r1097987240

Reply:
I might be thinking about this wrong, but I notice that the leap_year function itself is 10 lines long

Reply:
I tried changing max-module-lines to 10 and it indeed caused an error. I think max-module-lines at 15 is better.